.azure/pipelines/Phonebook.preview.yml

```diff
@@ -0,0 +1,193 @@
+pr: 
+  branches:
+    include:
+      - master
+
+variables:
+  image_repo: danielhabenicht/phonebook
+  image_tag: pr-$(System.PullRequest.PullRequestNumber) 
+  helm_artifact_name: HelmPhonebookPackage
+
+  jobs:
+    - job: build_phonebook_frontend
+      displayName: 'Phonebook Frontend'
+      pool:
+        vmImage: 'ubuntu-16.04'
+      steps:
+      - task: UseNode@1
+        inputs:
+          version: '11.x'
+
+      - script: node version.js $(image_tag) $(Build.SourceVersion) $(Build.SourceVersion)
+        displayName: 'Write Version Number'    
+        workingDirectory: 'Phonebook.Frontend/'
+
+      - script: docker build -t $(image_repo):$(image_tag) .
+        displayName: 'Build Image'              
+        workingDirectory: 'Phonebook.Frontend/'
+      
+      - script: |
+          docker login -u $DOCKER_REGISTRY_USER -p $(DOCKER_REGISTRY_PASSWORD)
+          docker push $(image_repo):$(image_tag)
+        workingDirectory: 'Phonebook.Frontend/'
+        displayName: 'Push Image'                      
+        env:
+          DOCKER_REGISTRY_PASSWORD: $(DOCKER_REGISTRY_PASSWORD)
+          DOCKER_REGISTRY_USER: $(DOCKER_REGISTRY_USER)
+
+    - job: build_phonebook
```

Comment:
maybe `build_phonebook_backend` is a better name

Reply:
No it's the helm chart. So it is the main application if you want to install the Phonebook that's the main package

Reply:
add `helm` or `deployment scripts` to the name. ⛑